recipe/build.sh

```diff
@@ -2,7 +2,17 @@
 cd ${SRC_DIR}/src
 source ../source.apache
 
+if [[ "$target_platform" == osx-* ]]; then
```

Comment:
This pattern matching works?!

Reply:
The `target_platform` is set in yaml files e.g. https://github.com/conda-forge/hole2-feedstock/blob/45b11db2967499dff5afe5b3ad21b4d0fecfdabb/.ci_support/osx_64_.yaml#L17 if that's what you mean?

Reply:
I meant the `*` – but I may have misunderstood what it does. In any case, wasn't really relevant.